src/In.ProjectEKA.HipService/Discovery/Patient/Ranker/LastNameRanker.cs

```diff
@@ -0,0 +1,22 @@
+namespace In.ProjectEKA.HipService.Discovery.Patient.Ranker
+{
+    using Helper;
+    using HipLibrary.Patient.Model.Response;
+    using Matcher;
+    using Patient = Model.Patient;
+
+    public class LastNameRanker : IRanker<Patient>
+    {
+        public PatientWithRank<Patient> Rank(Patient patient, string lastName)
+        {
+            var diff = FuzzyNameMatcher.LevenshteinDistance(patient.LastName, lastName);
+            if (diff == 0)
+                return new PatientWithRank<Patient>(patient, RankBuilder.StrongMatchRank,
+                    MetaBuilder.FullMatchMeta(Match.LAST_NAME));
+            if (diff <= 2)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=4839081)